lib/Serverless.js

```diff
@@ -70,6 +71,12 @@ class Serverless {
       return this.pluginManager.run(this.processedInput.commands);
     }
 
+    // check if tracking is enabled (and track if it's enabled)
+    const serverlessPath = this.config.serverlessPath;
```

Comment:
Yes good point! Would like to store it near to Serverless so that the users file system is not polluted with "non-related files".